### DISCUSSION TITLE
Sending change type notices

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#607] [Q&A] [answered]
Is there a problem in sending, without the changes node, a file that has had some modification?

**Comment by YvesJo** [comment, marked as answer]
Hi,
For non-technical questions, please contact the helpdesk [ted@publications.europa.eu](mailto:ted@publications.europa.eu).
As a reminder a change may only apply on an already published notice, also it has to have its dedicated section (changes node to indicated the reason for the change, identify the notice sections that have changed ...)
